### DISCUSSION TITLE
"List releases" frequently returns 502 Bad Gateway

### DISCUSSION THREAD

**Discussion by astellingwerf** [community/community#39863] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

I use Renovate to keep my Docker images up to date. The one containing Java checks for releases in the adoptium/temurin17-binaries project, but that one appears to consistently fail to check of updates. The larger the per_page query param is, the bigger the odds seem that the request will fail.

An example of the failing request and response:
```
         "options": {
           "headers": {
             "user-agent": "RenovateBot/34.22.2 (https://github.com/renovatebot/renovate)",
             "accept": "application/vnd.github.v3+json",
             "authorization": "***********",
             "accept-encoding": "gzip, deflate, br"
           },
           "url": "https://api.github.com/repos/adoptium/temurin17-binaries/releases?per_page=100",
           "hostType": "github-releases",
           "username": "",
           "password": "",
           "method": "GET",
           "http2": false
         },
         "response": {
           "statusCode": 502,
           "statusMessage": "Bad Gateway",
           "body": {"message": "Server Error"},
           "headers": {
             "server": "GitHub.com",
             "date": "Wed, 23 Nov 2022 17:22:27 GMT",
             "content-type": "application/json",
             "content-length": "32",
             "etag": "\"637e5382-20\"",
             "vary": "Accept-Encoding, Accept, X-Requested-With",
             "x-github-request-id": "28A0:7B48:3A8D6B4:77A1057:637E56C2",
             "connection": "close"
           },
           "httpVersion": "1.1",
           "retryCount": 2
         }
```

Is there anything that can be done about this at the client or server end?

**Comment by ossanna16** [comment]
Hi there @astellingwerf and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
